src/main/java/noel/Noel.java

```diff
@@ -0,0 +1,55 @@
+package noel;
+
+import noel.exceptions.NoelException;
+import noel.helper.Parser;
+import noel.helper.Storage;
+import noel.helper.Tasklist;
+import noel.helper.Ui;
+
+public class Noel {
+
+    static String HELLO_MSG = " Hello! I'm Noel!\nWhat can I do for you?";
```

Comment:
As per coding standard, they encourage associated constants to have a common prefix. I see that they have a common suffix but i guess if we were to follow the standard given, maybe you can find some way to give them a common prefix instead?